### DISCUSSION TITLE
Component does not generate css or javascript

### DISCUSSION THREAD

**Discussion by em1le** [django-components/django-components#386] [Q&A]
Good afternoon,

Firstly i want to thank the contributors for this great library.

Secondly, following is my issue :

I have declared an anchor component as follow.

```python
@component.register("anchor")
class Anchor(component.Component):
    def get_context_data(self, content: str) -> Dict[str, Any]:
        return {
            "content": content,
        }

    template = """
        <a class="anchor-component" href="#">{{ content }}</span></a>
    """

    css = """
        .anchor-component { background: pink; }
    """

    js = """
        (function(){
            if (document.querySelector(".anchor-component")) {
                document.querySelector(".anchor-component").onclick = function(){ alert("Clicked anchor!"); };
            }
        })()
    """
```

I load it in my template as follow (i have set `component_tags` in templates builtin).

```html
{%.extends 'base.html' %}

{%.block main %}
{% component "anchor" content="Login" %}
{%.endblock %}
```

Now when my page is loaded, the style and javascript are not present.

<img width="1512" alt="Screenshot 2024-02-23 at 14 57 00" src="https://github.com/EmilStenstrom/django-components/assets/22121480/3f477b70-35a1-4ea6-b663-fba5b612fd97">

What did i miss ? 

Thanks in advance.

**Comment by EmilStenstrom** [comment]
Here's a full example of what you need to do to add a new component: https://github.com/EmilStenstrom/django-components?tab=readme-ov-file#use-the-component-in-a-template - Does it make sense?

**Reply by em1le** [reply]
Thanks for your feedback.

**Comment by em1le** [comment]
Basically i forgot to add the directory in my `STATICFILES_DIRS`
